x-pack/plugins/monitoring/common/constants.ts

```diff
@@ -477,7 +477,7 @@ export const ALERT_DETAILS = {
     paramDetails: {
       threshold: {
         label: i18n.translate('xpack.monitoring.alerts.shardSize.paramDetails.threshold.label', {
-          defaultMessage: `Notify when a shard exceeds this size`,
+          defaultMessage: `Notify when primary average shard size exceeds this value`,
```

Comment:
Same here, I'd just say "Notify when average shard size exceeds" -- we are operating on the idea that primary shards are what matter, and that replicas are duplicates.